builder/vsphere/clone/step_clone.go

```diff
@@ -49,18 +52,28 @@ type StepCloneVM struct {
 func (s *StepCloneVM) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
 	ui := state.Get("ui").(packer.Ui)
 	d := state.Get("driver").(*driver.Driver)
-	vmPath := fmt.Sprintf("%s/%s", s.Location.Folder, s.Location.VMName)
+	vmPath := path.Join(s.Location.Folder, s.Location.VMName)
 
 	vm, err := d.FindVM(vmPath)
-
-	if s.Force == false && err == nil {
-		state.Put("error", fmt.Errorf("%s already exists, you can use -force flag to destroy it", vmPath))
-		return multistep.ActionHalt
-	} else if s.Force == true && err == nil {
-		ui.Say(fmt.Sprintf("the vm/template %s already exists, but deleting it due to -force flag", vmPath))
-		err := vm.Destroy()
-		if err != nil {
-			state.Put("error", fmt.Errorf("error destroying %s: %v", vmPath, err))
+	if err != nil {
+		if _, ok := err.(*find.NotFoundError); !ok {
+			state.Put("error", fmt.Errorf("error looking up old vm: %v", err))
+			return multistep.ActionHalt
+		}
+	} else {
+		if s.Force {
+			ui.Say(fmt.Sprintf("the vm/template %s already exists, but deleting it due to -force flag", vmPath))
+
+			// power off just in case it is still on
+			vm.PowerOff()
```

Comment:
It looks `vm.PowerOff()` returns an error. Do we need to handle it?

Reply:
we don't care about the  poweroff error, we only care about the destroy error.